liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java

```diff
@@ -46,6 +46,8 @@ public class StandardChangeLogHistoryService extends AbstractChangeLogHistorySer
     protected static final String LABELS_SIZE = "255";
     protected static final String CONTEXTS_SIZE = "255";
 
+    public static final String MD5_COLUMN_SIZE = "68";
```

Comment:
How did you calculate this?

Reply:
SHA256 size is `64 `. So 64 + `:` + version up to 3 digits (thinking big here!)
Originally it was `35`: 32 for MD5  + `:` + 2 version up to 2 digits